config/env.js

```diff
@@ -72,7 +72,9 @@ function getClientEnvironment (publicUrl) {
         // For example, <img src={process.env.PUBLIC_URL + '/img/logo.png'} />.
         // This should only be used as an escape hatch. Normally you would put
         // images into the `src` and `import` them in code to get their paths.
-        PUBLIC_URL: publicUrl
```

Comment:
Usually I don't do this, I just edit `Flags.elm`,  `src/scripts/config.js` and `env-config.js.sample`

Reply:
If we store the secret key that way then it will be very easy to find it in the source code. I introduced the `GRAPHQL_SECRET` env variable to be able to add it manually during the build process.